routes/relationship.js

```diff
@@ -1,152 +1,202 @@
-let express = require('express');
+let express = require("express");
 const fs = require("fs");
 let router = express.Router();
-const crypto = require('crypto');
-
+const crypto = require("crypto");
+const { client } = require("./dbhandler");
 /**
  * Create endpoint, which takes two node parameters to link together, as well as relationship info.
  */
-router.post('/create', function (req, res, next) {
-    // Grab the parameters from the request body that we need
-    const {node1, node2, relationship} = req.body;
-
-    // Ensure node1, node2, and relationship are specified
-    if (!node1 || !node2 || !relationship) {
-        res.status(400).send("Must specify node1, node2, and relationship!");
-        return;
-    }
-
-    // This assumes the file already exists (and it should)
-    let json = JSON.parse(fs.readFileSync(__dirname + "/../public/relationships.json", 'utf-8'));
-
-    let id = crypto.randomUUID();
-
-    // Add to JSON
-    json.push({id, node1, node2, relationship});
-
-    // Write new changes to file
-    fs.writeFile(__dirname + "/../public/relationships.json", JSON.stringify(json), function (err) {
-        if (err) {
-            res.status(400).send("Issue writing file!");
-        }
+router.post("/create", function (req, res, next) {
```

Comment:
Great work on the code connecting to Mongo and modifying a collection! We're changing the database format, so that the relationships (and nodes) are lists as part of a map document. So the relationship route should 1. expect the mapid that the relationship belongs to and 2. adds/removes/updates from that list of relationships in the map document. If you're curious about how this looks, take a peek at the dummy maps that got put in the 'blurp' database (collection is called 'maps').